### ISSUE TITLE
Detect square brackets in the wrong place

### ISSUE THREAD

**Issue by hbokh** [warpnet/salt-lint#141]
Hi there!
`salt-lint` is not detecting this:
```
package-name:
  pkg.installed: []
    - fromrepo: {{ grains['oscodename'] }}-backports
```

The square brackets should not be there.
Would it be possible to fix this behaviour, @roaldnefs ? 🤗
